control/control_input_merger/README.md

```diff
@@ -0,0 +1,21 @@
+**#Control input merger - merging control input form all active controllers**
+
+Subscribes on topics:
+ -surge_input
+ -sway_input
+ -heave_input
+ -roll_input
+ -pitch_input
+ -yaw_input
+All of message type "Wrench" which consist of forces and tourqe in x,y,z
+
+
+Publishes  on topic:
+ -/manta/thruster_manager/input , -> Wrench message
+
+**#Run**
+
+Run the depth hold node by using the command:
```

Comment:
should this say "Run the control input merge" instead of "Depth hold"?

Reply:
That is correct, updated readme have been pushed.

Reply:
Great :+1: